cmd/jaeger/internal/exporters/storageexporter/exporter.go

```diff
@@ -14,13 +14,16 @@ import (
 	"go.uber.org/zap"
 
 	"github.com/jaegertracing/jaeger/cmd/jaeger/internal/extension/jaegerstorage"
+	ch "github.com/jaegertracing/jaeger/plugin/storage/clickhouse"
 	"github.com/jaegertracing/jaeger/storage/spanstore"
 )
 
 type storageExporter struct {
-	config     *Config
-	logger     *zap.Logger
-	spanWriter spanstore.Writer
+	config             *Config
+	logger             *zap.Logger
+	spanWriter         spanstore.Writer
+	exportTraces       func(ctx context.Context, td ptrace.Traces) error
```

Comment:
nit: I would call this `consumeTraces`, because the argument that we pass to the exporter helper is `consumer.ConsumeTracesFunc`